pkg/kmm/preflightvalidationsocp.go

```diff
@@ -0,0 +1,290 @@
+package kmm
```

Comment:
nit: I think the file name should be preflightvalidationocp.go

Reply:
There were some places where this was at plural. Still nice catch !

Thanks , Updated